### PR TITLE
(#457) - Moving bitwise operations in bytes atom

### PR DESCRIPTION
**Pull request by IngeniariusSoftware**

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#458](https://github.com/cqfn/eo/issues/458); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by yegor256** [comment]
@IngeniariusSoftware I don't think you are doing it right. Try this test:

```
[] > conjuction-of-bytes
  02-EF-D4-05-5E-78-3A > a
  12-33-C1-B5-5E-71-55 > b
  a.and b > c
  c.eq
    ??-??-??-??-??-??-??
```

My point is that `bytes` may not contain only 8 bytes, but any number of them.

**Comment by IngeniariusSoftware** [comment]
@yegor256 , I fixed

File: eo-runtime/src/main/java/EOorg/EOeolang/EObytes$EOleft.java [hunk 1]
```diff
@@ -25,23 +25,29 @@
...
         super(sigma);
         this.add("x", new AtFree());
-        this.add("φ", new AtComposite(this, self -> new Data.ToPhi(
-            new Dataized(self.attr("ρ").get()).take(Long.class)
-            <<
-            new Dataized(self.attr("x").get()).take(Long.class)
-        )));
+        this.add("φ", new AtComposite(this, self -> {
+            final byte[] array = new Dataized(
+                    self.attr("ρ").get()
```
Comment (yegor256) [inline eo-runtime/src/main/java/EOorg/EOeolang/EObytes$EOleft.java:43]: @IngeniariusSoftware just curious, why do you use 8 spaces in your code if everywhere in the code we use 4?

File: eo-runtime/src/main/java/EOorg/EOeolang/EObytes$EOnot.java [hunk 2]
```diff
@@ -25,23 +25,26 @@
...
+    public EObytes$EOnot(final Phi sigma) {
         super(sigma);
-        this.add("x", new AtFree());
-        this.add("φ", new AtComposite(this, self -> new Data.ToPhi(
-            new Dataized(self.attr("ρ").get()).take(Long.class)
-            &
-            new Dataized(self.attr("x").get()).take(Long.class)
-        )));
+        this.add("φ", new AtComposite(this, self -> {
+            final byte[] array = new Dataized(
```
Comment (yegor256) [inline eo-runtime/src/main/java/EOorg/EOeolang/EObytes$EOnot.java:41]: @IngeniariusSoftware no, wait, here you use 4 and the next line it's 8. why?)

**Comment by yegor256** [comment]
@IngeniariusSoftware looks good, but I'm a bit concerned about your indentation policy. It's a bit messed up. See my comments above.

**Comment by 0crat** [comment]
Job [#458](https://github.com/cqfn/eo/issues/458) is already in scope

**Comment by IngeniariusSoftware** [comment]
@yegor256 , corrected, maybe something wrong with my indentation settings in IntelliJ

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#458](https://github.com/cqfn/eo/issues/458); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Pull request merged**